go/vt/schemadiff/table.go

```diff
@@ -1858,7 +1858,7 @@ func (c *CreateTableEntity) validate() error {
 		err := sqlparser.Walk(func(node sqlparser.SQLNode) (kontinue bool, err error) {
 			switch node := node.(type) {
 			case *sqlparser.ColName:
-				referencedColumns = append(referencedColumns, node.Name.String())
+				referencedColumns = append(referencedColumns, node.Name.Lowered())
```

Comment:
The list here is later used in error messages, so this means that in the error message we lose the proper casing. I think we need to move this to the map lookup (and take that approach in general then).